### PR TITLE
Migrate to postgresql

### PR DESCRIPTION
**Pull request by EnKarin**
test execution time with mysql ~ 1 minute
with postgresql ~ 38 seconds

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -17,19 +17,20 @@ spring:
...
                     CharSet: utf-8
                     useUnicode: true
+                default_schema: bookcrossing
         open-in-view: false
     datasource:
-        url: jdbc:mysql://localhost:3306/bookcrossing
-        username: root
-        password: root
-        driver-class-name: com.mysql.cj.jdbc.Driver
+        url: jdbc:postgresql://localhost:5432/bookcrossing
```
Comment (mfvanek) [inline src/main/resources/application.yml:30]: Could you please provide a docker compose file or a docker command in **readme.md** to make postgresql database up and running?

File: src/test/resources/db/migration/V1__init.sql [hunk 2]
```diff
@@ -1,24 +1,22 @@
...
     type integer,
     user_id integer not null,
     primary key (confirmation_mail)
 );
 
-create table t_attach (
+create table if not exists bookcrossing.t_attach (
     attach_id integer not null,
-    data longblob,
+    data oid,
```
Comment (mfvanek) [inline src/test/resources/db/migration/V1__init.sql:14]: Why did you decide to change column type?
`oid` in PG is numeric object identifier https://www.postgresql.org/docs/current/datatype-oid.html
`longblob` in MySQL is a binary large object https://tableplus.com/blog/2019/10/tinyblob-blob-mediumblob-longblob.html

**Review by mfvanek — CHANGES_REQUESTED** [review]

File: src/test/resources/db/migration/V1__init.sql (hunk 2 above)
Comment (EnKarin) [reply src/test/resources/db/migration/V1__init.sql:14]: Since pg by default stores large files in pg_largeobject and returns their oid

Comment (EnKarin) [reply src/test/resources/db/migration/V1__init.sql]: annotation @Lob indicated that the data would be large, so it was saved as large_object

**Review by mfvanek — APPROVED** [review]

**Pull request merged**